client/src/App.js

```diff
@@ -1,15 +1,57 @@
 import { Route, Switch } from 'react-router-dom';
-import { useState } from 'react';
+import { useState, useEffect } from 'react';
 import Home from './pages/Home';
 import CreateMentor from './pages/CreateMentor';
 import SearchMentor from './pages/SearchMentor';
+import Favourites from './pages/Favourites';
 import Header from './components/Header';
 import Navigation from './components/Navigation';
+import loadFromLocal from './lib/loadFromLocal';
+import saveToLocal from './lib/saveToLocal';
 
 
 function App() {
 
+  const MENTORS_KEY = 'mentorsList'
+  const FAVOURITES_KEY = 'favouritesList'
+
   const [open, setOpen] = useState(false);
+  const [mentorsApi, setMentorsApi] = useState(loadFromLocal(MENTORS_KEY) ?? []);
```

Comment:
I would consider renaming `mentorsApi` to simply `mentors`, `setMentorsApi` accordingly.